pages/_app.js

```diff
@@ -1,77 +1,48 @@
-import '../styles/globals.css'
-import '../styles/globals.scss'
+import '../styles/globals.css';
+import '../styles/globals.scss';
 import React from 'react';
-import App from 'next/app'
-import Link from 'next/link'
+import App from 'next/app';
+import Link from 'next/link';
 import { useEffect } from 'react';
-import {serviceName} from '../helpers/constants';
+import Header from '../compoments/header';
 
-function MyApp({ Component, pageProps, err  }) {
+function MyApp({ Component, pageProps, err }) {
   const enableJavascript = () => {
-    window.GOVUKFrontend.initAll()
-  }
-  useEffect(enableJavascript, [])
+    window.GOVUKFrontend.initAll();
+  };
+  useEffect(enableJavascript, []);
 
-  return <>
-    <header className="govuk-header " role="banner" data-module="govuk-header">
-      <div className="govuk-header__container govuk-width-container">
-        <div className="govuk-header__logo">
-          {/* eslint-disable-next-line @next/next/no-html-link-for-pages */}
-          <a href="https://www.lincoln.gov.uk/" className="govuk-header__logotype" target="_blank" rel="noopener noreferrer"/>
-        </div>
-        <div className="govuk-header__content">
-          <Link href="/">
-            <a
-              className="govuk-header__link govuk-header__link--service-name">
-              {serviceName}
-            </a>
+  return (
+    <>
+      <Header></Header>
+      <div className="govuk-width-container">
+        <main className="govuk-main-wrapper govuk-!-padding-0">
+          <Component {...pageProps} err={err} />
+        </main>
+      </div>
+      <footer className="govuk-footer " role="contentinfo">
+        <div className="govuk-width-container ">
+          <Link className="footer-custom-link" href="">
+            Accessibility
```

Comment:
Can content be changed to 'Accessibility Statement' to add a bit more context.

Can these footer links also include `govuk-footer__link` class attribute so they're styled as such - think that can replace the `footer-custom-link`class